frontend/next.config.js

```diff
@@ -1,13 +1,16 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
 	output: 'standalone',
-	reactStrictMode: true,
+	reactStrictMode: false, // Moved to _app.tsx, otherwise providers are called twice
```

Comment:
Added a comment why I did this: the websocket was connecting and instantly disconnecting.
It seems like `<React.StrictMode />` is causing `useEffect(..., [])` to trigger twice.
So I disabled strict mode here and moved it to _app.tsx.